src/app/core/common-components/entity-select/entity-select.component.ts

```diff
@@ -226,10 +230,53 @@ export class EntitySelectComponent<
       (e) => !e.isActive && this.autocompleteFilter(e),
     ).length;
   }
+
+  createNewEntity = async (input: string): Promise<E> => {
+    if (this._entityType?.length < 1) {
+      return;
+    }
+    if (this._entityType.length > 0) {
+      this.logger.warn(
+        "EntitySelect with multiple types is always creating a new entity of the first listed type only.",
+      );
+      // TODO: maybe display an additional popup asking the user to select which type should be created?
+    }
```

Comment:
This always logs an warning, is this intended? Should be ```(this._entityType.length > 1)``` or am i wrong?

Reply:
oh, yes!

```suggestion
    if (this._entityType.length > 1) {
      this.logger.warn(
        "EntitySelect with multiple types is always creating a new entity of the first listed type only.",
      );
      // TODO: maybe display an additional popup asking the user to select which type should be created?
    }
```